__init__.py

```diff
@@ -593,6 +593,8 @@ def handle_language_back_event(self, message):
     @killable_event(msg="pairing.backend.menu.stop")
     def handle_backend_menu(self):
         if not self.settings["enable_backend_selection"]:
+            if not is_paired():
+                self.handle_no_backend_selected(None)
```

Comment:
this will need an extra check in this handler also, it calls STT unconditionally 

```
        # continue to STT
        self.handle_stt_menu()
 ```

Reply:
There's a check in `handle_stt_menu()`; maybe we can avoid checking twice though..